src/bot/discord-helper.service.ts

```diff
@@ -12,6 +12,8 @@ import {
   TextChannel,
 } from 'discord.js';
 import { InjectDiscordClient, On } from '@discord-nestjs/core';
+import { InjectRepository } from '@mikro-orm/nestjs';
+import { EntityRepository } from '@mikro-orm/core';
```

Comment:
Double check in other files if we're getting `EntityRepository` from /core or /postgres